src/operator/skipUntil.ts

```diff
@@ -18,59 +17,46 @@ import { subscribeToResult } from '../util/subscribeToResult';
  * @method skipUntil
  * @owner Observable
  */
-export function skipUntil<T>(notifier: Observable<any>): Observable<T> {
+export function skipUntil<T>(this: Observable<T>, notifier: Observable<any>): Observable<T> {
   return this.lift(new SkipUntilOperator(notifier));
 }
-
-export interface SkipUntilSignature<T> {
-  (notifier: Observable<any>): Observable<T>;
-}
-
 class SkipUntilOperator<T> implements Operator<T, T> {
   constructor(private notifier: Observable<any>) {
   }
-
   call(subscriber: Subscriber<T>, source: any): TeardownLogic {
     return source._subscribe(new SkipUntilSubscriber(subscriber, this.notifier));
   }
 }
-
 /**
  * We need this JSDoc comment for affecting ESDoc.
  * @ignore
  * @extends {Ignored}
  */
 class SkipUntilSubscriber<T, R> extends OuterSubscriber<T, R> {
-
   private hasValue: boolean = false;
   private isInnerStopped: boolean = false;
-
   constructor(destination: Subscriber<any>,
               notifier: Observable<any>) {
     super(destination);
     this.add(subscribeToResult(this, notifier));
   }
-
   protected _next(value: T) {
     if (this.hasValue) {
       super._next(value);
     }
   }
-
```

Comment:
Most definitely an artifact of the script, we can either tweak the script or tweak the files after the fact.   We can just wait to tweak those things to closer to when we want to merge these changes in.